### PR TITLE
[CLI V2] Custom User Data, GraphQL, and Incoming Webhooks

### PR DESCRIPTION
**Pull request by nlarew**
### Issue JIRA link:

- (DOCSP-14936): [CLI V2] Enable Custom User Data
- (DOCSP-14953): [CLI V2] Expose Data in a Collection
- (DOCSP-14957): [CLI V2] HTTP Service
- (DOCSP-14956): [CLI V2] Configure Service Webhooks

### Docs staging link (requires sign-in on MongoDB Corp SSO):

- [Enable Custom User Data](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/cli-grabbag/users/enable-custom-user-data)
- [Expose Data in a Collection](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/cli-grabbag/graphql/expose-data)
- [HTTP Service](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/cli-grabbag/services/http)
- [Configure Service Webhooks](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/cli-grabbag/services/configure/service-webhooks)

### Review Guidelines
[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/includes/steps-create-a-service-webhook-cli.yaml [hunk 1]
```diff
@@ -1,49 +1,55 @@
...
-  {+ui+}, or by running the following command from an authenticated instance of
-  :doc:`realm-cli </deploy/realm-cli-reference>`:
-
-  .. code-block:: shell
-
-     realm-cli export --app-id=<App ID>
+  To define an incoming webhook with {+cli-bin+}, you need a local copy of your
+  application's configuration files.
+  
+  To pull a local copy of the latest version of your app, run the following:
```
Comment (mongomoe) [inline source/includes/steps-create-a-service-webhook-cli.yaml:7]: Would it help to link to https://docs.mongodb.com/realm/config/#std-label-app-configuration. 

Maybe something like: " To define an incoming webhook with {+cli-bin+}, you need a local copy of your :ref:`application's configuration <link>` files."

File: source/includes/steps-define-custom-user-data-import-export.yaml [hunk 2]
```diff
@@ -1,106 +1,50 @@
...
-     git push origin <branch name>
-
-  Once the application successfully deploys, {+service+} begins to associate
-  custom data with users. When a user logs in, {+service-short+} automatically
-  queries the custom user data collection for a document where the
-  specified :guilabel:`User ID Field` contains the user's ID. If a
-  document matches, {+service-short+} exposes the data in the document in the
-  ``custom_data`` field of that user's :ref:`user object
-  <user-objects>`.
+  Once you've configured custom user in ``/auth/custom_user_data.json``, you can
```
Comment (mongomoe) [inline source/includes/steps-define-custom-user-data-import-export.yaml:44]:
```suggestion
  Once you've configured a custom user in ``/auth/custom_user_data.json``, you can
```
Should it be 'a custom user' or 'the custom user'

**Comment by mongomoe** [comment]
LGTM, but can we do something about the width of the first column (labelled "action") in https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/cli-grabbag/services/http/#service-actions

**Review by mongomoe — APPROVED** [review]
LGTM 👍

File: source/includes/steps-define-custom-user-data-import-export.yaml (hunk 2 above)
Comment (nlarew) [reply source/includes/steps-define-custom-user-data-import-export.yaml:44]: Tweaked

File: source/includes/steps-create-a-service-webhook-cli.yaml (hunk 1 above)
Comment (nlarew) [reply source/includes/steps-create-a-service-webhook-cli.yaml:7]: Yup totally - I'll add this in a pass to all of the procedures

**Pull request merged**